tdvt/tdvt/tdvt.py

```diff
@@ -223,16 +223,16 @@ def enqueue_single_test(args, ds_info, suite):
         test_set = SingleExpressionTestSet(suite, get_root_dir(), args.expression_pattern, args.tds_pattern,
                                            args.test_pattern_exclude, ds_info)
 
-    test_config = TdvtTestConfig(from_args=args)
-    test_config.suite_name = suite
-    test_config.timeout_seconds = ds_info.timeout_seconds
-    test_config.logical = test_set.is_logical_test()
-    test_config.d_override = ds_info.d_override
-    test_config.run_as_perf = ds_info.run_as_perf
-    test_config.tds = test_set.tds_name
-    test_config.config_file = test_set.config_name
+    tdvt_invocation = TdvtInvocation(from_args=args)
+    run_time_config = RunTimeTestConfig()
 
-    return test_set, test_config
+    run_time_config.timeout_seconds = ds_info.timeout_seconds
+    run_time_config.d_override = ds_info.d_override
+    run_time_config.run_as_perf = ds_info.run_as_perf
+    run_time_config.tds = test_set.tds_name
+    test_set.run_time_config = run_time_config
```

Comment:
My linter throws an error for lines 232 & 233 - does `RunTimeTestConfig` have a `tds` attr? Also, I think `SingleLogicalTestSet` and `SingleExpressionTestSet` (or their parents) need to have `run_time_config` attrs added?